tests/test_wsireader.py

```diff
@@ -9,6 +9,22 @@
 import cv2
 from click.testing import CliRunner
 
+# -------------------------------------------------------------------------------------
+# Constants
+# -------------------------------------------------------------------------------------
+
+NDPI_TEST_TISSUE_BOUNDS = (30400, 11810, 30912, 12322)
```

Comment:
Looks fine to me, but I cannot see if you are using these in the file somewhere.